### ISSUE TITLE
Excessive space between the PMC install command and the Documentation section

### ISSUE THREAD

**Issue by karann-msft** [NuGet/NuGetGallery#4857]
The amount of whitespace seems excessive. Can we make it consistent with the space we have between say the Documentation and Dependencies sections.

![image](https://user-images.githubusercontent.com/16904420/31678686-bdb30baa-b323-11e7-920f-ab03b7192034.png)

**Comment by chenriksson** [comment]
Cause is ".tag-content { height: 79px; }" in bootstrap-theme.css.

**Comment by loic-sharma** [comment]
This is intentional. Third-party tabs, like `Paket CLI`, have a banner below them that provides a support link. First-party tabs do not have this notice. Without this additional space below first-party tabs, switching between first-party and third-party tabs would be a jarring experience as the whole page would shift. This additional space below first-party tabs was requested by @anangaur 

@chenriksson That should be `tab-content`, correct?

**Comment by chenriksson** [comment]
@loic-sharma yes, that was a typo. Screenshot below with the third party tab banner:

![image](https://user-images.githubusercontent.com/15364949/31728915-75ba7c2e-b3e2-11e7-8eb5-b7a249f3be65.png)

@karann-msft - closing as by design, but reopen if you think there's something we can do better here.

**Comment by anangaur** [comment]
We plan to put a line there for other tabs  as well to direct to installation on how to install packages.

**Comment by karann-msft** [comment]
I don't agree with the resolution. Let's discuss this.

**Comment by skofman1** [comment]
@karann-msft , please schedule a meeting to discuss this.

**Comment by karann-msft** [comment]
My proposal is to optimize for the case where there is no banner i.e. by default the highlighted space should be the same as that between the Documentation and Dependencies. I am ok if this space is reduced when a banner does appear. In no circumstance should the page jump when switching between tabs (which doesn't happen today either but stating for clarity)
